mlir/test/Dialect/Linalg/vectorization.mlir

```diff
@@ -419,6 +419,31 @@ module attributes {transform.with_named_sequence} {
 
 // -----
 
+// CHECK-LABEL: func @test_vectorize_unpack
+func.func @test_vectorize_unpack(%0 : tensor<7x1136x16x16xf32>) -> tensor<100x18176xf32>  {
+    // CHECK %[[c0:.*]] = arith.constant 0 : index
+    // CHECK: %[[tr0:.*]] = vector.mask %[[m0:.*]] {{.*}} vector.transfer_read %{{.*}} : tensor<7x1136x16x16xf32>, vector<7x1136x16x16xf32> } : vector<7x1136x16x16xi1> -> vector<7x1136x16x16xf32>
```

Comment:
This does not look right. If the provided vector sizes are [2, 4]. It should return something like `vector<2x4x16x16xf32>` (or something else), but not the whole shape.

Also please you can add a test with dynamic shapes but with static inner tile sizes.

Reply:
Yes you are right. When I implemented using `vectorSizes` this fixed itself.